lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -865,3 +865,54 @@ def test_course_catalog_access_num_queries(self, user_attr_name, action, course_
         course_overview = CourseOverview.get_from_id(course.id)
         with self.assertNumQueries(num_queries, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
             bool(access.has_access(user, action, course_overview, course_key=course.id))
+
+
+class CCXOverviewTestCase(SharedModuleStoreTestCase):
+    """
+    Tests for ccx course overviews
```

Comment:
nit: It's true that the "courselike" object we're testing here is a CourseOverview object, but it's a bit confusing to use the word "overview" here.  Could you change this to just "CCX courses" and remove the reference to Overview in the class and comment?